papaparse.js

```diff
@@ -1113,8 +1113,8 @@
 						header = header.trim();
 					}
 
-					if (_config.stripHeaders) {
-						header = header.replace(/\s/g,'');
+					if (_config.headerTransform) {
```

Comment:
Probably make this check for callability rather than truthiness? `typeof _config.headerTransform === 'function'`